September Serenity Updates (#1006)
akeneo/pim-helpcenter: content/updates/2021-09/6-tailoredexport-csvproductmodel.md

---
pim_announcement_img: ./img/TailoredExport_Create-export-profile.png
pim_announcement_alt_img: "Tailored Exports"
pim_announcement_audience:
- EE
- CE
---

> # Tailored Export Extended Capabilities

Reviewer: ```suggestion
# Tailored Exports Extended Capabilities
```

::: meta-data type="New" features="Productivity" available="mid-September" in="EE"

> Tailored Export now offers the same possibilities as any other export profile. Define the format, CSV or XLXS, and choose between exporting products or product models. With these extended capabilities, Tailored Export is about to become a must-have in your Akeneo PIM toolbox.

Reviewer: ```suggestion
Tailored Exports now offers the same possibilities as any other export profile. Define the format, CSV or XLXS, and choose between exporting products or product models. With these extended capabilities, Tailored Export is a must-have in your Akeneo PIM toolbox.
```
